.github/workflows/release.yml

```diff
@@ -34,7 +34,8 @@ jobs:
     name: Publish
     runs-on: ubuntu-latest
     needs: tag
-    if: needs.tag.outputs.tag != ''
+
+    if: github.event_name == 'workflow_dispatch'
```

Comment:
This changes the release flow of the SDK is a critical way. Please do this change in another PR, I don't agree with this strategy initially

Reply:
done